app/uk/gov/nationalarchives/omega/editorial/views/editSetRecordEdit.scala.html

```diff
@@ -203,6 +203,19 @@
 )
 )
 
+@govukTextarea(Textarea(
```

Comment:
Can this block be indented similarly to the others, and the closing parentheses each put on a closing line?

I know that scalafmt doesn't format twirl files...

Reply:
Agreed, it would have to be IJ reformatting, or manually.

Reply:
Done